src/tests/e2etest/CommandRunnerTests.cs

```diff
@@ -1,19 +1,279 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
 using NUnit.Framework;
 using System;
 using System.Text.Json;
 using System.Threading.Tasks;
 
 namespace E2eTesting
 {
-    public class CommandRunnerTests : E2eTest
+
+    [TestFixture]
+    public abstract class E2ETest
```

Comment:
Why is this class defined in CommandRunnerTests.cs instead of E2ETest.cs?

Reply:
The plan of action for now is to try out these changes with the CommandRunner E2E tests first, since this is where we are seeing most of the timeout issues, then bring these changes to the `E2ETest.cs` and refactor the other E2E tests to use this new proposed base class.

Reply:
Why not just try them with all of them and not just the CommandRunner tests?